rock_gazebo.orogen

```diff
@@ -49,10 +49,17 @@ task_context "ModelTask" do
     # The covariance matrix to be used as velocity covariance. Default is unset (NaN)
     property 'cov_velocity', '/base/Matrix3d'
 
+    # The acceleration of the model w.r.t. the world frame
+    output_port 'ang_accel_samples', '/base/samples/RigidBodyAcceleration'
```

Comment:
angular_acceleration_samples is fine.

Yes it's strange, the linear_acceleration could also be an output from the link. I actually planned to implement it but ended up dragging the implementation.

The Advanced Navigation INS that we will use give us an angular acceleration (it's possibly a dirty derivative of the angular velocity), and we use this information to "cancel" the model on our control. From the IMU sensor on gazebo we can get the angular velocity with noise, so we implemented this feature to get the angular acceleration from gazebo and them we would add a noise.

About the RigidBodyAcceleration, it would be good to have one RigidBodyAcceleration with linear and angular.